src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -706,6 +709,8 @@ export class SuggestWidget implements IContentWidget, IListVirtualDelegate<IComp
 	}
 
 	showSuggestions(completionModel: CompletionModel, selectionIndex: number, isFrozen: boolean, isAuto: boolean): void {
+		this.preferDocPositionTop = false;
+
```

Comment:
Shouldnt we reset `this.docsPositionPreviousWidgetY` here as well?

Reply:
Updated :)